srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml

```diff
@@ -875,6 +875,12 @@ SRTP packets                 <------->      SRTP packets
         allocation of even values over odd ones until the even code
         points are consumed to avoid conflicts with pre standard
         versions of EKT that have been deployed.</t>
+
+        <t> All new EKT message MUST be defined to have a length as
+        second from the last element so that implementations that do
+        not understand the new extension can still skip over it when
+        parsing the message. <t>
```

Comment:
following your advice of "it is what is" and less explanation of the design, I will drop the 2nd half of the sentence